lib/geordi/chromedriver_updater.rb

```diff
@@ -94,7 +105,7 @@ def chromedriver_url(chrome_version)
         if chromedriver && chromedriver["url"]
           chromedriver["url"]
         else
-          Interaction.fail("Could not find chromedriver download url for Chrome version v#{chrome_version}.")
+          raise ProcessingError, "Could not find chromedriver download url for Chrome version v#{chrome_version}."
```

Comment:
Please change all output to speak of "Chrome $version", e.g. "Chrome 119.2344.1"